keras/src/quantizers/quantizers.py

```diff
@@ -127,6 +130,348 @@ def get_config(self):
         }
 
 
+@keras_export("keras.quantizers.adjust_and_nudge_quantization_range")
```

Comment:
Could there be a shorter / friendlier name than `adjust_and_nudge_quantization_range`?

Also, does it need to be a public facing API?